experiments/kompanion/cmd/export/export.go

```diff
@@ -50,20 +50,22 @@ const (
 	kompanion export --include-namespaces=my-team
 
 	# target only specific namespace prefixes AND specific object prefixes
-	kompanion export --target-namespaces=my-team --target-objects=important
+	kompanion export --target-namespaces=my-team --target-objects=logging
 	`
 )
 
 var ExportCmd = &cobra.Command{
 	Use:     "export",
+	Short:   "export Config Connector resources",
 	Example: examples,
 	RunE:    runE,
 	Args:    cobra.ExactArgs(0),
 }
 
 const (
 	// flag names.
-	kubeconfigFlag = "kubeconfig"
+	kubeconfigFlag       = "kubeconfig"
+	reportNamePrefixFlag = "report-prefix"
```

Comment:
These do not need to be constants.  It just makes the flags harder to read, and you aren't repeating them.